cwr/README.md

```diff
@@ -0,0 +1,79 @@
+# CWR META Indexer
+
+The `cwr` package implements a META indexer for a
+[CWR Files Format ](http://members.cisac.org/CisacPortal/consulterDocument.do?id=26603) and exposes
+a GraphQL API for querying the index.
+
+## Overview
+
+This module convert CWR transactions ,as appears in CWR formatted files, to META objects and index these with
+certain keys.
+
+The `Converter` type is used to read a CWR file , convert to META objects and append to a META stream.
+It is done in 2 phases:
+Phase 1 -
+  Convert CWR format files to standard JSON files (CWR-JSON).
+  This is done using [CWR-DataApi python project](http://cwr-dataapi.readthedocs.io/en/latest/)
+  There is an [updated fixed forked version](https://github.com/orenyodfat/CWR-DataApi) as a git submodule in this project.  
```

Comment:
Have the changes in the fork been pull requested upstream? We should be avoiding the maintenance of forks.